src/Core/src/PublicAPI/net6.0-windows10.0.19041/PublicAPI.Shipped.txt

```diff
@@ -1938,6 +1938,7 @@ override Microsoft.Maui.Handlers.ViewHandler<TVirtualView, TPlatformView>.SetupC
 override Microsoft.Maui.Handlers.WebViewHandler.ConnectHandler(Microsoft.UI.Xaml.Controls.WebView2! platformView) -> void
 override Microsoft.Maui.Handlers.WebViewHandler.CreatePlatformView() -> Microsoft.UI.Xaml.Controls.WebView2!
 override Microsoft.Maui.Handlers.WebViewHandler.DisconnectHandler(Microsoft.UI.Xaml.Controls.WebView2! platformView) -> void
+override Microsoft.Maui.Handlers.WindowHandler.ConnectHandler(Microsoft.UI.Xaml.Window! platformView) -> void
```

Comment:
so one shouldn't add this line right ? this should not change

Reply:
i made a mistake in the initial commit. I removed it instead of adding the REMOVED version